clang/docs/ReleaseNotes.rst

```diff
@@ -59,6 +59,9 @@ Clang Python Bindings Potentially Breaking Changes
 - Calling a property on the `CompletionChunk` or `CompletionString` class
   statically now leads to an error, instead of returning a `CachedProperty` object
   that is used internally. Properties are only available on instances.
+- `TranslationUnitLoadError` now contains an error code in `error_code`
+  attribute. Also, `TranslationUnit.reparse` will throw `TranslationUnitLoadError`
+  when operation fails.
```

Comment:
Code highlighting should be in double ` (yes the example above that did it incorrectly... will fix)